providers/openshift/authentication.go

```diff
@@ -74,11 +74,10 @@ type ClientCertAuthenticationOptions struct {
 	ClientCA string
 }
 
-func (s *ClientCertAuthenticationOptions) AddFlags(fs *flag.FlagSet) {
-	fs.StringVar(&s.ClientCA, "client-ca-file", s.ClientCA, ""+
-		"If set, any request presenting a client certificate signed by one of "+
-		"the authorities in the client-ca-file is authenticated with an identity "+
-		"corresponding to the CommonName of the client certificate.")
+func (s *ClientCertAuthenticationOptions) SetClientCAFile(file string) {
+	if len(file) > 0 {
+		s.ClientCA = file
```

Comment:
Since `s.ClientCA` will the empty string already, you can remove the if statement.